miso-web/src/main/webapp/pages/editSample.jsp

```diff
@@ -521,7 +565,47 @@
           </table>                               
         </div>
       </c:if>
-      
+
+      <c:if test="${sampleCategory eq 'Tissue Processing' or sampleCategory eq 'new'}">
```

Comment:
should only be included if `${sampleCategory eq 'Tissue Processing'}` since you can't "receive" a Tissue Processing at this time, and the 'single' mode of this page cannot be used for propagation